backend/src/test/java/de/bund/digitalservice/ris/norms/application/service/ElementServiceTest.java

```diff
@@ -0,0 +1,119 @@
+package de.bund.digitalservice.ris.norms.application.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import de.bund.digitalservice.ris.norms.application.port.input.LoadElementFromNormUseCase;
+import de.bund.digitalservice.ris.norms.application.port.output.LoadNormPort;
+import de.bund.digitalservice.ris.norms.domain.entity.Norm;
+import de.bund.digitalservice.ris.norms.utils.NodeParser;
+import de.bund.digitalservice.ris.norms.utils.XmlMapper;
+import java.util.Optional;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class ElementServiceTest {
+  final LoadNormPort loadNormPort = mock(LoadNormPort.class);
+
+  final ElementService service = new ElementService(loadNormPort);
+
+  @Nested
+  class loadElementFromNorm {
+    @Test
+    void itLoadsAnElementFromANorm() {
+      // Given
+      var eli = "eli/bund/bgbl-1/2000/s1/1970-01-01/1/deu/regelungstext-1";
+      var eid = "meta-1";
+
+      var normXml =
```

Comment:
How about using a fixture file (just for reasons of readability)?

Reply:
IMO this is small enough not to use a fixture